lib/bitcoin/lock/LockMonitor.ts

```diff
@@ -277,12 +296,8 @@ export default class LockMonitor {
     latestSavedLockInfo: SavedLockModel,
     desiredLockAmountInSatoshis: number): Promise<boolean> {
 
-    const currentBlockTime = await this.bitcoinClient.getCurrentBlockHeight();
-
-    console.info(`Current block: ${currentBlockTime}; Current lock's unlock block: ${currentValueTimeLock.unlockTransactionTime}`);
-
     // Just return if we're not close to expiry

```

Comment:
The old logic was testing if we are close, now we are testing if we are exactly at release time right?

Reply:
Good point. Will change.